include/cscore.h

```diff
@@ -1,12 +1,12 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#ifndef CSCORE_H_
-#define CSCORE_H_
+#ifndef INCLUDE_CSCORE_H_
+#define INCLUDE_CSCORE_H_
```

Comment:
From the Google style guide: `To guarantee uniqueness, they should be based on the full path in a project's source tree.` so the -I directives are irrelevant. The header guards should completely disambiguate regardless of the installation configuration, and the rule presented in the style guide does that. That includes the case where, for some reason, all the headers are installed into one folder.